### ISSUE TITLE
Cannot use required images with Avatar

### ISSUE THREAD

**Issue by johnf** [GeekyAnts/NativeBase#4863]
### Description

Avatar no longer supports required images

### CodeSandbox/Snack link

https://snack.expo.dev/yZSTh2TeX

### Steps to reproduce

1. See snack above the first Avatar doesn't have an image

### NativeBase Version

3.3.10

### Platform

- [X] Android
- [ ] CRA
- [ ] Expo
- [X] iOS
- [ ] Next

### Additional Information

This was broken by #4723 as imported images don't have a URI attribute.

**Comment by mayank-96** [comment]
Hello @johnf, Thanks for reporting this issue, we'll fix it soon.

**Comment by mayank-96** [comment]
Hello @johnf, you can try the following syntax to get it working.
```
     <Avatar bg="amber.500" source={{ uri: image }}>
           TS
      </Avatar>
```

**Comment by johnf** [comment]
@mayank-96 that doesn't work for me. I get an error about casting URI from Double to String.

**Comment by johnf** [comment]
This has been fixed by changes in v3.3.11
Thanks!
